src/main/java/com/mraof/minestuck/client/util/MSKeyHandler.java

```diff
@@ -72,7 +74,10 @@ public static void onKeyInput(InputEvent event)	//This is only called during the
 		
 		while(captchaKey.isPressed())
 		{
-			if(!Minecraft.getInstance().player.getHeldItemMainhand().isEmpty())
+			if(!Minecraft.getInstance().player.getHeldItemMainhand().isEmpty() && !ModList.get().isLoaded("cosmeticarmorreworked"))
+				MSPacketHandler.sendToServer(CaptchaDeckPacket.captchalogue());
+			//Apparently when CosmeticArmorReloaded is active, the game no longer refuses to call onKeyInput when a gui is open, and so we need to make sure that a gui is not open.
```

Comment:
You think you can explain why that is happening?

Reply:
No. I have no clue why it happens, it just does.